### DISCUSSION TITLE
Should I run SCTranform on each demultiplexed sample individually?

### DISCUSSION THREAD

**Discussion by weishwu** [satijalab/seurat#9435] [General]
My data has 2 10x runs and each multiplexed 5 replicates. I wonder whether I should run SCTranform on each pool (so 2 models in total) before demultiplexing, or run demultiplexing first and then run SCTransform on each individual replicate (10 models in total).

Also, I see almost no batch effect between the demultiplexed replicates in each pool. Does it make sense to do batch correction only between the two pools instead of between all 10 samples? If so, do I technically have to run SCTransform on each pool? (If I run SCTransform on each replicate, there will be 10 models so there is no way to tell JoinLayers to only correct batch effect between the two pools).

**Comment by samuel-marsh** [comment]
Hi,

See other comment on your recent post about converting issue to discussion.  On topic of this question though it is going to depend on your experiment and examination of the data.  Sequencing batch and/or 10X lane batch are both possible batch effects but it is also possible that each sample has its own batch effect before being pooled.  There is not right answer to this question for every dataset.  Again as I mentioned in the other post integration/splitting layers should only be used in analysis when needed and use of integration where not required may actually harm analysis rather than help.

Best,
Sam

**Comment by weishwu** [comment]
Thanks @samuel-marsh for your answer.
My main question here is to ask whether I should run SCTransform on each demultiplexed sample individually, or on the pool before demultiplexing. Is it also a data-dependent choice without being absolutely right or wrong?
